src/app/common/components/StatusIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { Card } from '@rebass/emotion';
+import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
+import theme from '../../../theme';
+import * as React from 'react';
+
+interface IProps {
+  status: string;
+}
+
+const StatusIcon: React.FunctionComponent<IProps> = ({
```

Comment:
Look into updating names for StatusIcon/ any component with redundant "Status" name in it to be more descriptive